metricbeat/module/vsphere/datastorecluster/_meta/fields.yml

```diff
@@ -17,4 +17,12 @@
       type: long
       description: >
         Total free space on this storage pod, in bytes.
-      format: bytes
\ No newline at end of file
+      format: bytes
+    - name: datastore.names
+      type: keyword
+      description: >
+        List of all the datastore names.
```

Comment:
```suggestion
        Names of all the datastores in the Datastore Cluster
```

Reply:
I think there are so many variations we have done for all the metricsets.
We should keep it same format for all of them. which one should be current one?
```
List of all the Datastore names associated with the cluster.
List of all the VM names.
List of all the datastore names.
Names of the hosts connected to this network.
Names of virtual machines on the resource pool.
Names of the datastore associated to this virtualmachine.
```

Reply:
The association part is important to state in the description. Rest I am ok with any of it, whatever we have used the most in the metricsets.

Reply:
Then I will go with this one, it seems more appropriate.
`List of all the Datastore names associated with the cluster.`

Reply:
List of all the Datastore names associated with the `datastore` cluster.
We shouldn't make it confusing with the cluster metricset.